### DISCUSSION TITLE
How shitty are school Chromebooks for you

### DISCUSSION THREAD

**Discussion by nathblade16** [3kh0/ext-remover#1048] [Polls JOIN TITANIUM NETWORK TO SHARE PLEASE]
Please comment some weird things that have happend to your school chromebook or chromebooks in general
for me, I charged my lenovo chromebook using a lenovo charger, biggest mistake of my life
it got fried and my IT guy says he would fix it a while ago and gave me a newer chromebook until then, never heard back since

**Comment by S-PScripts** [comment]
I'm not sure myself what happened to mine but one day it bricked. I was charging it and after a bit when I thought it was probably full I opened and it showed an error. It said something about the verification of the OS (it happened a long time ago) and I had to go to the IT office and get another until they repaired it. I had tried using the Chrome utility recovery extension by putting a USB with the recovery stuff into my Chromebook, but after finishing, it returned to showing the same error. Restarting wouldn't remove the error. It might be that it updated to v117 (which was apparently a buggy update that got pulled) whilst it was charging but who knows...

**Reply by nathblade16** [reply]
yeah, chromebooks are shit, also remember not to charge a chromebook with a chromebook charger

**Reply by nathblade16** [reply]
what happend to mine was I was charging it and coding some scratch stuff while it was charging, it shut off randomly with no warning, I smelled smoke and it never turned on again

**Reply by Guslight4** [reply]
I once dropped it from 10 meters above the ground but it was fine then 9 days later i dropped from 30 centimeters and its screen smashed

**Reply by nathblade16** [reply]
> I once dropped it from 10 meters above the ground but it was fine then 9 days later i dropped from 30 centimeters and its screen smashed

ok but I bought a scamazon water tablet and it still has not arrived yet, can you help with that

**Reply by Guslight4** [reply]
you just need to wait 4 years because that is the delivery time and the cancel time is 1 year so you got scamazoned

**Reply by nathblade16** [reply]
> you just need to wait 4 years because that is the delivery time and the cancel time is 1 year so you got scamazoned

I will contact your customer support email, I might be able to get something through

**Comment by S-PScripts** [comment]
> yeah, chromebooks are shit, also remember not to charge a chromebook with a chromebook charger

I got my Chromebook from the school (like most) and they gave the chargers. I don't have any other chargers for my Chromebook with the same port 📦

**Reply by nathblade16** [reply]
its a joke from personal experience

**Comment by nathblade16** [comment]
bruh, 35% of people said yes, the blocker, this was meant to be a skid filter
